scripts/automation/cortex-master.sh

```
@@ -1,194 +1,194 @@
-            10) audit_repo ;;
-            0) echo "Goodbye!"; exit 0 ;;
-            *) echo "Invalid option" ;;
-        esac
-        
-        echo ""
-        read -p "Press Enter..." 
-    done
-}
-
-main
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)